### ISSUE TITLE
Enable user namespaces in the Debian package

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#5502]
Lots of users are having problems (e.g. https://twitter.com/robdavies1978/status/1154709097184411648) installing Brave on Debian because of the extra step of enabling user namespaces.

Maybe we should do it for them by shipping a `/etc/sysctl.d/brave.conf` file containing:

    kernel.unprivileged_userns_clone = 1

and then doing a `sysctl -p /etc/sysctl.d/brave.conf` in the postinst script.

**Comment by fmarier** [comment]
Perhaps we should do this in `brave-keyring` to avoid having to patch the Chromium source.

**Comment by fmarier** [comment]
Fixed in the `brave-keyring` 1.5 package.

**Comment by tbergeron** [comment]
@fmarier could that new package update have caused this:

```
[ anon@xps ~  ] sudo apt upgrade
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  brave-keyring
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
Need to get 3,496 B of archives.
After this operation, 0 B of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 https://brave-browser-apt-release.s3.brave.com disco/main amd64 brave-keyring all 1.5 [3,496 B]
Fetched 3,496 B in 0s (11.6 kB/s)
(Reading database ... 266210 files and directories currently installed.)
Preparing to unpack .../brave-keyring_1.5_all.deb ...
Unpacking brave-keyring (1.5) over (1.4) ...
Setting up brave-keyring (1.5) ...
OK
sysctl: cannot stat /proc/sys/kernel/unprivileged_userns_clone: No such file or directory
```
just curious, I cannot figure out how to fix this and am wondering if it's normal? thanks!

**Comment by fmarier** [comment]
Yes, we had a problem with version 1.4 of the package. This should be fixed in 1.5 though. Try `apt update && apt upgrade`.
